packages/react/src/patterns/sub-patterns/Layout/__stories__/Layout.stories.js

```diff
@@ -44,111 +47,115 @@ storiesOf('Patterns (Sub-Patterns)|Layout', module)
 
         <p style={{ paddingBottom: '1rem' }}>Layout component begins below.</p>
 
-        <Layout
-          type={select('Layout Type (type)', type, type['1-4 - 3/4 layout'])}
-          marginTop={select(
-            'Top Margin (marginTop)',
-            layoutOptions,
-            layoutOptions['layout-03']
-          )}
-          marginBottom={select(
-            'Bottom Margin (marginBottom)',
-            layoutOptions,
-            layoutOptions['layout-06']
-          )}
-          stickyOffset={number('Sticky offset (in pixels)', 0)}>
-          <div
-            data-sticky={select(
-              'Sticky left column',
-              ['true', 'false'],
-              'true'
+        <div className={nested ? 'bx--grid' : ''}>
```

Comment:
To test whether it nests properly it needs to fit inside of a `bx--grid` `bx--row` and some sort of `bx--col` like `bx--col-lg-12`.

Reply:
currently, it is inside a `bx--col-lg-12`